src/components/Spinner/index.jsx

```diff
@@ -0,0 +1,20 @@
+import { PureComponent } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+@withStyles({
+  center: {
+    textAlign: 'center',
+  },
+})
+export default class Spinner extends PureComponent {
+  render() {
+    const { classes } = this.props;
+
+    return (
+      <div className={classes.center}>
+        <CircularProgress thickness={7} />
```

Comment:
This is a little bit too thick relative to the font size. Maybe we can go with thickness 5.